content/news/2024-??-??-bevy-0.15/index.md

```diff
@@ -33,7 +33,14 @@ Thank you so much to [everyone who helped out](https://discord.com/channels/6910
 The features above may be great, but what else does Bevy have in flight?
 Peering deep into the mists of time (predictions are _extra_ hard when your team is almost all volunteers!), we can see some exciting work taking shape:
 
-- TODO
+- **Bevy Scene Notation:** With required components, Cart's [master plan](https://github.com/bevyengine/bevy/discussions/14437) for BSN is starting to come together. Over the next few month, he's going to be heads-down developing a Bevy-specific file format (complete with matching macro and IDE support), the `Construct` trait (to easily include asset data in scenes), patches (to layer modifications to scenes) and experimenting with approaches to reactivity for UI.
+- **Better font support:** While `cosmic_text` is a huge leap forward for text shaping and rendering, our approach to handling fonts and type-faces is still quite crude. Bidirectional text, working with system fonts, a convenient Markdown-style "bold this section of the text" API, font fallback and more are planned.
```

Comment:
Markdown-style feels like a poor way to describe it to me. Maybe say "a more expressive API for text styling like bolding/colors(?)/etc" or something? Unsure

Reply:
I think that "more expressive" is part of it, but a markup / Markdown flavored way to write lightly formatted text is high on our wish list.